integrations/langchain/tests/utils/chat_models.py

```diff
@@ -1,7 +1,8 @@
-import pytest
-
 from typing import Generator
 from unittest import mock
+
```

Comment:
Should we update these tests to also assert that the tool description + args description are properly set? Lmk if it's already done and I missed it (just looking at changes since my last review rn)

Reply:
Added a new test `test_vector_search_retriever_tool_description_generation` to explicitly test these changes.